packages/polka/src/index.ts

```diff
@@ -81,7 +83,10 @@ export default class Polka implements IModule, IServerHandler {
     const polkaConfig = config?.get<PolkaConfig>(this.name);
 
     const app = polka({
-      onError: (err, _req, res) => {
+      onError: (err, req, res) => {
+        const traceId = req.traceId || uuidv4();
+        err.traceId = traceId;
```

Comment:
This is not `type-safe` . Any reason why we are doing this?

Reply:
No reason - what should I do differently?

Reply:
If `no reason` why do it?